File: leo2-node/src/main/java/org/deku/leo2/node/data/repositories/HolidayctrlRepository.java
package org.deku.leo2.node.data.repositories;

import org.deku.leo2.node.data.entities.HolidayCtrl;
import org.deku.leo2.node.data.entities.HolidayCtrlPK;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QueryDslPredicateExecutor;

/**
 * Created by JT on 15.05.15.
 */
public interface HolidayctrlRepository extends JpaRepository<HolidayCtrl, HolidayCtrlPK>, QueryDslPredicateExecutor {
}

File: leoz-node/src/main/java/org/deku/leoz/node/data/repository/master/HolidayCtrlRepository.kt
package org.deku.leoz.node.data.repository.master

import org.deku.leoz.node.data.jpa.MstHolidayCtrl
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.querydsl.QuerydslPredicateExecutor

/**
 * Holiday repository
 * Created by masc on 15.05.15.
 */
interface HolidayCtrlRepository :
        JpaRepository<MstHolidayCtrl, Long>,
        QuerydslPredicateExecutor<MstHolidayCtrl>
